backend-spec/README.md

```diff
@@ -0,0 +1,102 @@
+
+# Example Shunter Application Back End Specification
+
+If a back end doesn't exist in the language or framework of your choice, feel free to create one. Your application will need to comply with this specification.
+
+
+## Naming
+
+If you want your back end to be housed under this GitHub organisation, you'll need to name it using the following pattern. If you're using a framework, you should include this in the repo name as well.
+
+```
+example-backend-<language>
+example-backend-<language>-<framework>
+```
+
+Examples:
+
+```
+example-backend-ruby
```

Comment:
maybe I should switch them, I wanted to demonstrate that you _can_ write framework-free back ends. Shall I change to:

```
example-backend-node
example-backend-ruby-sinatra
```